Formula/s/sshfs.rb

```diff
@@ -9,8 +9,6 @@ class Sshfs < Formula
     sha256 x86_64_linux: "a98d273e64706971684935a3ae87da16b1dda98f7289eb79e82f4cdfb7f12bb8"
   end
 
-  deprecate! date: "2023-01-01", because: :repo_archived
-
```

Comment:
Does this no longer apply? https://github.com/libfuse/sshfs?tab=readme-ov-file#development-status

Reply:
sounds fair, but the repo is no longer archived? Let me ask upstream to check with maintenance status.